src/condor_includes/KeyCache.h

```diff
@@ -64,13 +74,14 @@ class KeyCacheEntry {
 
     char *               _id;
     condor_sockaddr*              _addr;
-    KeyInfo*             _key;
+	std::vector<KeyInfo*> _keys;
```

Comment:
Switch from raw pointers to `std::unique_ptr` here as it captures the memory ownership design of the interface.